### DISCUSSION TITLE
Sending deep learning model through EdgeX services

### DISCUSSION THREAD

**Discussion by anhtuan85** [edgexfoundry/community#311] [Q&A] [answered]
Hi everyone.
I have a question: Can I send deep learning model file (with format like: .h5, .tflite) to EdgeX device services?

**Comment by cloudxxx8** [comment, marked as answer]
You can send the binary file via REST Device Service
Define your Device Profile, and send the binary file like the image file in the example:
https://docs.edgexfoundry.org/3.1/microservices/device/services/device-rest/howto/Testing/

**Reply by anhtuan85** [reply]
Yes, thank you so much.
